src/main/java/fr/ospiea/oscra/user/object/User.java

```diff
@@ -56,12 +145,102 @@ public void setPhoneNumber(String phoneNumber) {
         this.phoneNumber = phoneNumber;
     }
 
-    public String getEmail() {
-        return email;
+    public Date getBirthday() {
+        return birthday;
     }
 
-    public void setEmail(String email) {
+    public void setBirthday(Date birthday) {
+        this.birthday = birthday;
+    }
+
+    public String getPosition() {
+        return position;
+    }
+
+    public void setPosition(String position) {
+        this.position = position;
+    }
+
+    public String getFixNumber() {
+        return fixNumber;
+    }
+
+    public void setFixNumber(String fixNumber) {
+        this.fixNumber = fixNumber;
+    }
+
+    public Civility getCivility() {
+        return civility;
+    }
+
+    public void setCivility(Civility civility) {
+        this.civility = civility;
+    }
+
+    protected User() {}
+/*
+    public User(String username, String firstName, String lastName, String email, String password, boolean enabled,
+                Role role, String roadNumber, String road, String supplementaryAddress, String postalCode, String city,
+                String phoneNumber, Date birthday, String position, String fixNumber, Civility civility) {
+        this.username = username;
+        this.firstName = firstName;
+        this.lastName = lastName;
         this.email = email;
+        this.password = password;
+        this.enabled = enabled;
+        this.role = role;
+        this.roadNumber = roadNumber;
+        this.road = road;
+        this.supplementaryAddress = supplementaryAddress;
+        this.postalCode = postalCode;
+        this.city = city;
+        this.phoneNumber = phoneNumber;
+        this.birthday = birthday;
+        this.position = position;
+        this.fixNumber = fixNumber;
+        this.civility = civility;
     }
+*/
```

Comment:
delete unused comment. Next time could you check your PR yourself in the first time to avoid me of writing words about deleting comment? Thank you in advance.